src/components/ha-cover-controls.html

```diff
@@ -0,0 +1,60 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
+
+<link rel="import" href="../util/cover-model.html">
+
+<dom-module id="ha-cover-controls">
+  <template>
+    <style>
+      .state {
+        text-align: right;
+        white-space: nowrap;
+        width: 127px;
```

Comment:
The controls are right aligned because `state-card-cover` has `class='horizontal justified layout'`.

`width:127px` does nothing because the "natural" width of the 3 buttons is 126.5px. When some buttons are unsupported they are `visibility: hidden` so width is not changed.

If the `state` container were wider, for example `226.px` then  `text-align: right;` would affect to which side of the 226.5px the 3 buttons are aligned. Since the container tightly wraps the buttons - `text-align: right;` has no effect.